tests/pw/feature-map/feature-map.yml

```
@@ -92,12 +92,14 @@
       vendor:
           vendor can view product menu page [lite]: true
           vendor can view add new product page [lite]: true
+          vendor can view add new product page [lite]: true
```

yamllint

[error] duplication of key "vendor can view add new product page [lite]" in mapping

(key-duplicates)